.github/workflows/build-libs.yml

```diff
@@ -30,29 +30,27 @@ jobs:
       - name: Install build dependencies
         run: |
           sudo apt-get update
-          sudo apt-get install build-essential git ocaml ocaml-nox ocaml-native-compilers camlp5 libgtk2.0 libgtksourceview2.0 liblablgtk-extras-ocaml-dev ocaml-findlib libnum-ocaml-dev emacs
+          sudo apt-get install coq
+          type coqc
+          coqc --version
 
-      # Change into UniMath directory and build UniMath
       - name: Build UniMath and libraries TypeTheory, Largecatmodules, and SetHITs
+        ### we don't use "make install", because the user-contrib directory might be write-protected
         run: |
-          pushd $GITHUB_WORKSPACE
-          make build-coq
           make -j2
-          make install
-          export PATH=$PATH:$PWD/sub/coq/bin/
-          echo $PATH
-          popd
-          # build TypeTheory
+	  : set the Coq path
```

Comment:
This looks like a tab instead of space, which is a syntax error in this kind of file.